dashboard/app/models/course_offering.rb

```diff
@@ -445,6 +449,18 @@ def high_school_level?
     grade_levels_list.any? {|g| HIGH_SCHOOL_GRADES.include?(g)}
   end
 
+  def find_corresponding_offerings_for_pl_course
```

Comment:
This will also be useful for showing which courses to encourage users to assign to sections.